_posts/daiblej/2020-09-02-custom-turtle-daiblej

```diff
@@ -0,0 +1,9 @@
+---
+layout:post
+author:daiblej
+title:"Janet's custom turtle"
```

Comment:
You'll need some spaces after the `:` here. This is because a `:` and a space are what separate the field name from the data in these YAML headers. Without the space, our blog can't tell what's data and what's its field name.

Reply:
Thanks for your comment. I think now the code is correct.